python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -246,6 +246,48 @@ def test_append_index(a, b):
     assert_eq(expected.index, actual.index)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        {"a": [1, 2]},
+        {"a": [1, 2, 3], "b": [3, 4, 5]},
+        {"a": [1, 2, 3, 4], "b": [3, 4, 5, 6], "c": [1, 3, 5, 7]},
+        {"a": [np.nan, 2, 3, 4], "b": [3, 4, np.nan, 6], "c": [1, 3, 5, 7]},
+        {1: [1, 2, 3], 2: [3, 4, 5]},
+        {"a": [1, None, None], "b": [3, np.nan, np.nan]},
+        {1: ["a", "b", "c"], 2: ["q", "w", "u"]},
+        {1: ["a", np.nan, "c"], 2: ["q", None, "u"]},
+        pytest.param(
+            {},
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/11080"
+            ),
+        ),
+        pytest.param(
+            {1: [], 2: [], 3: []},
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/11080"
+            ),
+        ),
+        pytest.param(
+            [1, 2, 3],
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/11080"
+            ),
+        ),
+    ],
+)
+def test_axes_dataframe(data):
```

Comment:
We're already in dataframe.py, so this naming is redundant.
```suggestion
def test_axes(data):
```

Reply:
To be clear, I completely realize that our tests don't necessarily follow this right now, but I'm in the process of pushing out new standards and best practices for tests and would like to start enforcing them.